lib/rbtrace/cli.rb

```diff
@@ -407,6 +415,10 @@ def self.run
           tracer.puts "=> #{res}"
         end
 
+      elsif opts[:interactive_given]
+        require "rbtrace/interactive/#{opts[:interactive]}"
+        load `which #{opts[:interactive]}`.chomp
```

Comment:
irb does work, it's just very slow therefore might not be very useful. However, I just tried to merge your new master, and it seems irb works much better now! It's still a bit slow but it's much faster now.

As for the other repl, in the case of this pull request, `rib`, which tab completion does work (I don't know how it works though, but it's implemented via [bond](https://github.com/cldwalker/bond)), and backtrace colourizing also works, session history, and lots of other plugins also work (they're mostly very simple).

I won't say those features are critical (well, tab completion could be, which irb doesn't work, but rib does work, and I assume it's not impossible to get it irb work, could be tough but not impossible), but they're still useful.

I am also ok if rbtrace just implements its own repl, since it's better than nothing and I could understand why we should keep things simple. However a plugin system which people could still pick whichever repl they prefer would be awesome. A lot of features are still working :)